packages/openapi-types-ghes-3.1/types.d.ts

```diff
@@ -4448,6 +4448,534 @@ export interface paths {
      */
     post: operations["apps/create-content-attachment"];
   };
+  "/app/hook/deliveries": {
+    /** This endpoint does not exist ghes-3.1.json. It was added in api.github.com.json */
+    get: operations["apps/list-webhook-deliveries"];
+  };
```

Comment:
We have to remove all paths with `x-octokit.diff[version].type === 'removed'` so these new operation types are not being added, or we have to make sure that `@octokit/types` can handle it. Which ever is simpler right now.

This will all be obsolete once the work is done on https://github.com/octokit/octokit-next.js